File: python_matlab_comparison/scripts/cpu/time_sweep_cpu.m
%% run all matlab timing

file_dir = fileparts(mlfilename('fullpath')); %get the directory of this file
pycom_path = fullfile(file_dir,'../../'); %assume the base is 2 levels up from this

DEBUG = false;

fprintf("Running Basic Sweep");
time_sweep_basic_cpu;
fprintf("Running Extended Sweep");
time_sweep_extended_cpu;
fprintf("Running Matrix Sweep");
time_sweep_matrix_cpu;
fprintf("Running Sparse Sweep");
time_sweep_sparse_cpu;
fprintf("Running FDFD Sweep");
time_sweep_fdfd_cpu;
fprintf("Running Beamforming Sweep");
time_sweep_beamforming_cpu;

File: pycom/timing/scripts/cpu/time_sweep_cpu.py
# -*- coding: utf-8 -*-
"""
Created on Mon Dec 23 20:09:23 2019

@author: aweiss
"""
#run each of these scripts by importing them

DEBUG=False
output_directory = './data'

print("Running Basic Sweep")
import time_sweep_basic_cpu
print("Running Extended Sweep")
import time_sweep_extended_cpu
print("Running Matrix Sweep")
import time_sweep_matrix_cpu
print("Running Sparse Sweep")
import time_sweep_sparse_cpu
print("Running FDFD Sweep")
import time_sweep_fdfd_cpu
print("Running Beamforming Sweep")
import time_sweep_beamforming_cpu